app/models/notification.rb

```diff
@@ -2,8 +2,31 @@ class Notification < ActiveRecord::Base
   belongs_to :event
   # has_many :recipients
 
-  VALID_STATUSES = %w(pending active canceled complete expired)
-  validates :status, inclusion: { in: VALID_STATUSES }
+  enum status: [ :pending, :active, :canceled, :complete, :expired ]
 
+  before_create :validate_status
```

Comment:
https://github.com/ReadyResponder/ReadyResponder/issues/262 requires all created notifications to either be `pending`, `active` or `cancelled`.